Remotion/Validation/Core/DefaultValidatorExtensions.cs

```diff
@@ -30,10 +31,40 @@ public static class DefaultValidatorExtensions
       return ruleBuilder.SetValidator(p => new NotNullValidator(p.ValidationMessage));
     }
 
```

Comment:
I know that I'm still missing the individual tests in DefaultValidatorExtensionsTest. SInce we've now added some generic extension methods that could apply to the same type, could you check if we either have integration tests that cover this possible ambiguity or add those as unit tests? If it's too much work to actually make the tests green / add good assertions, it's enough for now to add the calls and keep the test as Ignored so the compiler can at least tell us that we did all right.

Reply:
I have added tests for all extension methods.